lib/route_translator/translator.rb

```diff
@@ -6,8 +6,6 @@
 module RouteTranslator
   module Translator
     class << self
```

Comment:
I intentionally moved private keyword from L9 to L19 to use it in `#define_generate_prefix`. Do you prefer adding new public method for `#define_generate_prefix` which does `yield`?